vars_example

```diff
@@ -0,0 +1,8 @@
+all:
```

Comment:
This file shouldn't be in the root of the repo, if it was even intended to be included at all.